src/core/flow.test.ts

```diff
@@ -241,30 +241,51 @@ describe('turn', () => {
     }
   });
 
-  describe('moveLimit', () => {
+  describe('minMoves', () => {
     test('without phases', () => {
       const flow = Flow({
         turn: {
-          moveLimit: 2,
+          minMoves: 2,
         },
       });
+
```

Comment:
“Stretch” goal here, if you have the time: would be good to wrap each block that ends with an `expect` assertion (or two) in its own `test`. Basically this formalises the comments you’ve added, so it’s more like:

```js
test('player 0 can’t end their turn when minMoves hasn’t been reached', () => {
  // ...
  expect(...);
});
```

There’ll also be a couple of tests that need updating, now that we are keeping `moveLimit` also setting `minMoves`.